src/logql.grammar

```diff
@@ -397,6 +414,10 @@ ConvOp {
     "`" ![`]* "`"
   }
 
+  ParserFlag {
+    "--strict" | "--keep-empty"
```

Comment:
Have you considered saving these as keywords? Just curious, I am not sure if that would be better.

Reply:
I tried to implement it by adding `--strict` as tokens, but I can't make it work. I was just curios, but this works as expected also for query such as `{label="value"} |= "--strict" | logfmt --strict` so LGTM.

Reply:
Question would be if we want to further distinguish the flags by lezer? E.g. any use cases where we need to know that it's a query with a `StrictParserFlag`?

Reply:
We could still do it by checking the value inside `ParserFlag` node.